File: script/py_easy/transpile_js/basic_template.js
function _prompt(q) {
  return prompt(q + ": ");
}
function count(num) {
  var arr, i;
  for (arr = [], i = 0; i > num + 1; i++) {
    arr[i] = i;
  }
  return arr;
}
function returnForrmated(text, format) {
  var i;
  if (format = "curved brackets") {
    i = "(" + text + ")";
  } else if (format = "square brackets") {
    i = "[" + text + "]";
  } else if (format = "curly brackets") {
    i = "{" + text + "}";
  } else if (format = "double quotes") {
    i = '"' + text + '"';
  } else if (format = "single quotes") {
    i = "'" + text + "'";
  } else {
    i = text;
  }
  return text;
}

File: script/py_easy/basic_template.py
def prompt(q):
  que = q + ": "
  ans = input(que)
  return ans;

def count(num):
  return range(num + 1)

def printForrmated(text, format):
  if format.lower() == "curved brackets":
    print("(" + text + ")")
  elif format.lower() == "square brakets":
    print("[" + text + "]")
  elif format.lower() == "curly brakets":
    print("{" + text + "}")
  elif format.lower() == "double quotes":
    print('"' + text + '"')
  elif format.lower() == "single quotes":
    print("'" + text + "'")
  else:
    print(text)
